openff/toolkit/tests/test_molecule.py

```diff
@@ -3887,14 +3894,15 @@ def test_perceive_residues_natoms_mainchain_histidine(self):
         assert unlabelled_counter == 12
         assert counter == offmol.n_atoms - 12
 
-    def test_perceive_residues_natoms_cyxteine(self):
+    @pytest.mark.parametrize("strict_chirality", (True, False))
```

Comment:
If these perceive residues tests were their own class (e.g. TestMoleculeResiduePerception), I think you could parametrize the whole class at once?

Reply:
Also, it would be nice to have a test where changing `strict_chirality` affected the outcome of `perceive_residues` :)

Reply:
Yes to both. Thanks! Will do that.